### ISSUE TITLE
Just launched my Ollie powered website + blog => A few items on my wish list

### ISSUE THREAD

**Issue by verquist** [OllieWP/ollie#51]
Hey Mike,

I love the Ollie theme. When I started building my new website, I first tried Spectra One. I wasn't happy with it. So I switched to Ollie. Wow! My website immediately felt 70% finished. I was off and running!

It's a beautiful theme. Simple. Elegant. Love the Patterns! Can't be happier. Here's my new website + blog. => https://engine2.us/

Just a few items on my wish list.

- Easy way to change fonts. => Don't get me wrong. The default fonts are beautiful. But I'd like to use Roboto 900 for all headers and Open Sans 400 for all body text. I love your thoughts on "Responsive control in WordPress." Ideally I could just override the default fonts in one place. And let you handle all the responsive adjustments elegantly.
- Styling for Contact Form 7 or similar => I ended up using "Ultimate Addons for Contact Form 7" plugin to do the styling. It works fairly well. But it's a pain. And not a perfect match with the Ollie stying.
- Easy way to remove rounded corners on Cover images. => I had to use custom css to override.

That's it. I built my Ollie powered website + blog in a few short weeks, and that's all I could think of for my wish list!

Thanks again,
Jim

**Comment by mikemcalister** [comment]
Thanks for this great feedback, Jim! Apologies for the late response. 

There are some changes coming in WP core that should help with choosing fonts. We also have some ideas about ways of providing curated fonts that you can choose from. 

If we had a simple, straight forward contact form for Ollie, would you use that? It's a bummer that customers always have to go searching for a contact form. And then you have to wrestle the styles. I'm curious if folks would use a simple contact form.

**Comment by verquist** [comment]
Hi Mike, This is great. Here are my thoughts.

Easy way to change fonts => Look forward to better support for changing fonts in the future! I'm sure you will come up with an elegant solution when WP core is up to the task. It's why I decided to not jump through hoops to use custom fonts right away.

Styling for Contact Form 7 or similar => Unfortunately, a simple, straight forward contact form for Ollie will not help me much. First, my website has two forms. Second, my website has a custom contact form (https://engine2.us/contact/). I may be an edge case. But I doubt it. I suspect you will have lots of people clamoring for various changes and customizations to your simple contact form, frustrating what you were trying to achieve. 

I think it would be better to just provide Ollie styling on the Contact Form 7 classes. Then you get all the benefits and flexibility that Contact Form 7 provides for free.

Honeypot traps => One unexpected benefit I discovered in using my own forms instead of pre-built forms from someone else is I added custom honeypot traps to both of my forms. Wow! Do they ever work!! I even added a honeypot trap to my blog's comments section. Yes, there are honeypot plugins. But there are big advantages in rolling your own custom traps. First, the way many honeypot traps work is stupid. E.g., disabling the submit button when honeypot trap is activated. This alerts the robots of the honeypot trap, which kind of defeats the purpose. Second, it's easy for robots to program around any particular honeypot scheme. But I don't see robots taking the time to create a workaround just for my website. So far this scheme has worked flawlessly for my forms + blog comments.

Thank you again, Mike. I love the Ollie theme. Look forward to seeing it get even better!

Feel free to reach out to me if you want to discuss specifics on any of these ideas. I have ideas. :-)
